src/components/VisualPanel/VisualPanel.module.scss

```diff
@@ -0,0 +1,41 @@
+
+.panelWrapper{
+    // width: 300px;
+    // height: 756px;
+    // margin: 30%;
+    display: flex;
```

Comment:
These lines seem to have no effect. I suggest removing it:
```
display: flex;
flex-direction: column;
```

Reply:
If I remove it, items will overflow instead of scroll. Panel is a flexbox which have title and list these two children.